repos/system_upgrade/el7toel8/actors/selinux/selinuxprepare/tests/component_test.py

```diff
@@ -68,7 +68,7 @@ def test_SELinuxPrepare(current_actor_context):
         api.current_logger().warning("Error listing SELinux customizations: %s", str(e.stderr))
 
     semodule_list = [SELinuxModule(name=module, priority=int(prio), content="", removed=[])
-                     for (prio, module) in test_modules + [["400", "permissive_abrt_t"]]]
+                     for (prio, module) in test_modules.reverse() + [["400", "permissive_abrt_t"]]]
```

Comment:
that wont' work
- test_modules.reverse() returns None while changing underlying list structure;
- you shouldn't modify global vars

Reply:
Sorry, should have been reversed(test_modules)